### ISSUE TITLE
Resources.designer.cs global::UXDivers.Artina.Player.Droid.Resource.Attribute.* missing

### ISSUE THREAD

**Issue by johannperez** [UXDivers/Gorilla-Player-Support#83]
Hi guys I've just downloaded GorillaPlayer in order to test it on Xamarin Studio at OSX and I am having some errors building the Player for Android. They are a lot but they all seem to be some definition missing for global::UXDivers.Artina.Player.Droid.Resource.Attribute.<something>

¿Did I miss something on the installation steps?

These are the errors:

```
	Resources/Resource.designer.cs(220,149): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Attribute' does not contain a definition for `mediaRouteSettingsDrawable'
	Resources/Resource.designer.cs(369,143): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Color' does not contain a definition for `design_textinput_error_color'
	Resources/Resource.designer.cs(477,146): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Dimension' does not contain a definition for `design_fab_content_size'
	Resources/Resource.designer.cs(487,160): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Dimension' does not contain a definition for `design_navigation_padding_top_default'
	Resources/Resource.designer.cs(500,143): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Dimension' does not contain a definition for `design_tab_min_width'
	Resources/Resource.designer.cs(501,148): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Dimension' does not contain a definition for `dialog_fixed_height_major'
	Resources/Resource.designer.cs(502,148): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Dimension' does not contain a definition for `dialog_fixed_height_minor'
	Resources/Resource.designer.cs(503,147): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Dimension' does not contain a definition for `dialog_fixed_width_major'
	Resources/Resource.designer.cs(504,147): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Dimension' does not contain a definition for `dialog_fixed_width_minor'
	Resources/Resource.designer.cs(510,163): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Dimension' does not contain a definition for `mr_media_route_controller_art_max_height'
	Resources/Resource.designer.cs(597,136): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Drawable' does not contain a definition for `ic_setting_dark'
	Resources/Resource.designer.cs(598,137): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Drawable' does not contain a definition for `ic_setting_light'
	Resources/Resource.designer.cs(609,140): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Drawable' does not contain a definition for `mr_ic_settings_dark'
	Resources/Resource.designer.cs(610,141): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Drawable' does not contain a definition for `mr_ic_settings_light'
	Resources/Resource.designer.cs(630,112): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Id' does not contain a definition for `art'
	Resources/Resource.designer.cs(634,116): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Id' does not contain a definition for `buttons'
	Resources/Resource.designer.cs(649,130): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Id' does not contain a definition for `default_control_frame'
	Resources/Resource.designer.cs(651,119): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Id' does not contain a definition for `disconnect'
	Resources/Resource.designer.cs(676,134): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Id' does not contain a definition for `media_route_control_frame'
	Resources/Resource.designer.cs(677,125): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Id' does not contain a definition for `media_route_list'
	Resources/Resource.designer.cs(678,134): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Id' does not contain a definition for `media_route_volume_layout'
	Resources/Resource.designer.cs(679,134): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Id' does not contain a definition for `media_route_volume_slider'
	Resources/Resource.designer.cs(689,119): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Id' does not contain a definition for `play_pause'
	Resources/Resource.designer.cs(694,119): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Id' does not contain a definition for `route_name'
	Resources/Resource.designer.cs(710,117): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Id' does not contain a definition for `settings'
	Resources/Resource.designer.cs(723,113): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Id' does not contain a definition for `stop'
	Resources/Resource.designer.cs(725,117): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Id' does not contain a definition for `subtitle'
	Resources/Resource.designer.cs(732,118): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Id' does not contain a definition for `title_bar'
	Resources/Resource.designer.cs(780,146): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Layout' does not contain a definition for `mr_media_route_chooser_dialog'
	Resources/Resource.designer.cs(781,160): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Layout' does not contain a definition for `mr_media_route_controller_material_dialog_b'
	Resources/Resource.designer.cs(782,141): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Layout' does not contain a definition for `mr_media_route_list_item'
	Resources/Resource.designer.cs(795,132): error CS0117: `UXDivers.Artina.Player.Droid.Resource.String' does not contain a definition for `ApplicationName'
	Resources/Resource.designer.cs(796,122): error CS0117: `UXDivers.Artina.Player.Droid.Resource.String' does not contain a definition for `Hello'
	Resources/Resource.designer.cs(816,158): error CS0117: `UXDivers.Artina.Player.Droid.Resource.String' does not contain a definition for `mr_media_route_button_content_description'
	Resources/Resource.designer.cs(817,149): error CS0117: `UXDivers.Artina.Player.Droid.Resource.String' does not contain a definition for `mr_media_route_chooser_searching'
	Resources/Resource.designer.cs(818,145): error CS0117: `UXDivers.Artina.Player.Droid.Resource.String' does not contain a definition for `mr_media_route_chooser_title'
	Resources/Resource.designer.cs(819,153): error CS0117: `UXDivers.Artina.Player.Droid.Resource.String' does not contain a definition for `mr_media_route_controller_disconnect'
	Resources/Resource.designer.cs(820,160): error CS0117: `UXDivers.Artina.Player.Droid.Resource.String' does not contain a definition for `mr_media_route_controller_no_info_available'
	Resources/Resource.designer.cs(821,148): error CS0117: `UXDivers.Artina.Player.Droid.Resource.String' does not contain a definition for `mr_media_route_controller_pause'
	Resources/Resource.designer.cs(822,147): error CS0117: `UXDivers.Artina.Player.Droid.Resource.String' does not contain a definition for `mr_media_route_controller_play'
	Resources/Resource.designer.cs(823,163): error CS0117: `UXDivers.Artina.Player.Droid.Resource.String' does not contain a definition for `mr_media_route_controller_settings_description'
	Resources/Resource.designer.cs(824,147): error CS0117: `UXDivers.Artina.Player.Droid.Resource.String' does not contain a definition for `mr_media_route_controller_stop'
	Resources/Resource.designer.cs(982,164): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Style' does not contain a definition for `RtlOverlay_Widget_AppCompat_ActionButton_Overflow'
	Resources/Resource.designer.cs(1261,162): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Styleable' does not contain a definition for `FloatingActionButton_android_background'
	Resources/Resource.designer.cs(1413,128): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Styleable' does not contain a definition for `Theme'
	Resources/Resource.designer.cs(1414,145): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Styleable' does not contain a definition for `Theme_actionBarDivider'
	Resources/Resource.designer.cs(1415,152): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Styleable' does not contain a definition for `Theme_actionBarItemBackground'
	Resources/Resource.designer.cs(1416,148): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Styleable' does not contain a definition for `Theme_actionBarPopupTheme'
	Resources/Resource.designer.cs(1417,142): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Styleable' does not contain a definition for `Theme_actionBarSize'
	Resources/Resource.designer.cs(1418,148): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Styleable' does not contain a definition for `Theme_actionBarSplitStyle'
	Resources/Resource.designer.cs(1419,143): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Styleable' does not contain a definition for `Theme_actionBarStyle'
	Resources/Resource.designer.cs(1420,149): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Styleable' does not contain a definition for `Theme_actionBarTabBarStyle'
	Resources/Resource.designer.cs(1421,146): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Styleable' does not contain a definition for `Theme_actionBarTabStyle'
	Resources/Resource.designer.cs(1422,150): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Styleable' does not contain a definition for `Theme_actionBarTabTextStyle'
	Resources/Resource.designer.cs(1423,143): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Styleable' does not contain a definition for `Theme_actionBarTheme'
	Resources/Resource.designer.cs(1424,149): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Styleable' does not contain a definition for `Theme_actionBarWidgetTheme'
	Resources/Resource.designer.cs(1425,146): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Styleable' does not contain a definition for `Theme_actionButtonStyle'
	Resources/Resource.designer.cs(1426,148): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Styleable' does not contain a definition for `Theme_actionDropDownStyle'
	Resources/Resource.designer.cs(1427,153): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Styleable' does not contain a definition for `Theme_actionMenuTextAppearance'
	Resources/Resource.designer.cs(1428,148): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Styleable' does not contain a definition for `Theme_actionMenuTextColor'
	Resources/Resource.designer.cs(1429,149): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Styleable' does not contain a definition for `Theme_actionModeBackground'
	Resources/Resource.designer.cs(1430,155): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Styleable' does not contain a definition for `Theme_actionModeCloseButtonStyle'
	Resources/Resource.designer.cs(1431,152): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Styleable' does not contain a definition for `Theme_actionModeCloseDrawable'
	Resources/Resource.designer.cs(1432,151): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Styleable' does not contain a definition for `Theme_actionModeCopyDrawable'
	Resources/Resource.designer.cs(1433,150): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Styleable' does not contain a definition for `Theme_actionModeCutDrawable'
	Resources/Resource.designer.cs(1434,151): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Styleable' does not contain a definition for `Theme_actionModeFindDrawable'
	Resources/Resource.designer.cs(1435,152): error CS0117: `UXDivers.Artina.Player.Droid.Resource.Styleable' does not contain a definition for `Theme_actionModePasteDrawable'

```

**Comment by johannperez** [comment]
Hi @LeoHere!

I am not even sure this is relevant, but I've just had a very similar problem with XLabs (the one they report here https://github.com/XLabs/Xamarin-Forms-Labs/issues/1125) 

It seems to be something related to the Xamarin.Forms version. My XF version is 2.2.0.31

Please let me know if I can help on any way.

**Comment by gc1508** [comment]
I am getting the same error. Any solution around this?

**Comment by HEBOS** [comment]
Same here.
Please advice us do we need to add some dlls to forms project too?

**Comment by LeoHere** [comment]
Hi guys, we are about to release a version that fix all these issues. Hold on please.

**Comment by donbing** [comment]
I see 0.9.0.0 on github, this seems to fix the build issue. but it crashes when performing the server handshake now. is there an updated server version somewhere?

**Comment by LeoHere** [comment]
No we didn't release it yet. In order to use those nuget packages you need the compatible gorilla desktop app, but we didn't release it yet. We will soon, hopefully this week.

**Comment by LeoHere** [comment]
This was fixed in 0.9.1.2
